checker/examples/units-extension/Frequency.java

```diff
@@ -6,6 +6,8 @@
 
 /**
  * Units of frequency, such as hertz (@{@link Hz}).
+ * Skip test below ensures the framework's automated tests does not consider this file as a test file.
```

Comment:
Why is this necessary? This is just an annotation, so I wouldn't expect an error in this file.